src/styles/01-colors/_colors-utility.scss

```diff
@@ -32,8 +32,8 @@ $warning-secondary: #ec7b1f;
 
 //Mapping where we associate a name string with each SASS var
 $colors: (
-  black: $black,
-  white: $white,
+  "black": $black,
```

Comment:
Can you explain why the quotes are here, in a comment

Reply:
@hokei I added a comment explaining this, to hopefully prevent someone from removing the quotes in the future.